tests/test_main.py

```diff
@@ -165,8 +165,8 @@ def test_exotic_image(
         resources / pdf,
         outfile,
         '-dc' if pytest.helpers.have_unpaper() else '-d',
-        '-v',
-        '1',
+        # '-v',
```

Comment:
Do you know why these were commented out?

Reply:
I have switched the default log level. I am going to revisit this once i have refactored to `logging`.